data/assets/scene/solarsystem/planets/earth/satellites/misc/tiangong.asset

```diff
@@ -23,7 +23,7 @@ local omm = asset.resource({
 local TiangongPosition = {
   Identifier = "TiangongPosition",
   Parent = transforms.EarthInertial.Identifier,
-  BoundingSphere = 54.5, -- half the width
+  BoundingSphere = 27.8, -- half the width
```

Comment:
I think it fits since I think it is a bug and it is related to models.

Reply:
Yeah but the fix of the bug becomes "invisible" if its part of this PR - especially since there is no issue for the bug to be closed. You can't see from the history that this was a problem before this PR. Do you get what I mean? 

If you don't want a separate commit for it, at least add a sentence to the description of this PR explaining that this issue is fixed :) For some traceability